runtime-interface/src/lib.rs

```diff
@@ -39,10 +39,10 @@ pub use sp_std::{convert::TryFrom, result::Result, vec::Vec};
 pub trait GearRI {
     /// Init lazy-pages.
     /// Returns whether initialization was successful.
-    fn init_lazy_pages() -> bool {
+    fn init_lazy_pages(pages_final_prefix: [u8; 32]) -> bool {
```

Comment:
You need to make separate function `init_lazy_pages` with #[version(2)]. And leave this one without changes. in `gear-lazy-pages` crate you need to support both versions: for old runtimes and for new runtimes

Reply:
Good catch, thanks! However we will drop everything after this PR, won't we? @breathx

Reply:
And vara?

Reply:
I think we will not drop vara, so you need to support.

Reply:
What do you mean `not active network`? If you mean no lazy-pages is used yet on vara - then it's not the point. Old runtimes require that all import functions are supported by binary node.